app/src/main/res/values-sw600dp-land/refs.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
w600dp != sw600dp, sw is constant shortest width regardless of orientation , w is the current width.